app/build.gradle

```diff
@@ -29,6 +29,8 @@ android {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
```

Comment:
kindly update this version
use  - `implementation 'androidx.recyclerview:recyclerview:1.1.0'`